packages/synapse-react-client/src/components/TeamSubjectsSelector/TeamBadgeOrError.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useMemo } from 'react'
+import { useGetTeamList } from '../../synapse-queries'
+import { Skeleton, Stack, Typography } from '@mui/material'
+import { ErrorTwoTone } from '@mui/icons-material'
+import TeamBadge from '../TeamBadge'
+
+export type TeamBadgeOrErrorProps = {
+  teamId: string
+}
+
+export const TeamBadgeOrError: React.FunctionComponent<
+  TeamBadgeOrErrorProps
+> = (props: TeamBadgeOrErrorProps) => {
+  const { teamId } = props
+
+  const { data: teamList, isLoading, error } = useGetTeamList([teamId])
```

Comment:
Why useGetTeamList instead of useGetTeam?

Reply:
I don't recall a good reason, so I'll switch to using `useGetTeam` instead!